packages/blade/src/components/Badges/_decisions/decisions.md

```diff
@@ -0,0 +1,61 @@
+# Badge <!-- omit in toc -->
+Badges are used to show small amount of color coded metadata, which are ideal for getting users attention.
+
+<img src="./badge-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Badge](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A417)
+
+
+## API
+
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `positive`,`negative`, `notice`, `information`, `neutral` | No | `neutral` | The variant of the badge to be rendered. |
+| contrast | `low`, `high` | No | `low` | The contrast of the badge to be rendered. |
+| size | `medium`, `small` | No | `medium` | The size of the badge to be rendered. |
+| children | `string` | **Yes** | undefined | The label to be rendered within the badge. |
+| icon | `Icon` | No | undefined | The Blade `Icon` component to be rendered for the badge. |
+| labelStyle | `italic`, `normal` | No | `normal` | The font style of the badge's label. |
+| labelWeight | `bold`, `regular` | No | `regular` | The font weight of the badge's label. |
+
+### Sample Usage
+```jsx
+import { Badge } from '@razorpay/components';
+
+<Badge 
+  variant='positive'
+  contrast='high'
+  icon={InfoIcon} 
+  size='large' 
+  labelStyle='italic' 
+  labelWeight='bold'
+>
+  New
+</Badge>
+```
+
+## Open Questions
+#### Dev
+- Should the label be automatically capitalized when `labelWeight` is `bold`
+- Should we call it `labelWeight`, `labelStyle` or `fontWeight`, `fontStyle` or `textWeight`, `textStyle`?
```

Comment:
I like `fontStyle` and `fontWeight` as it is similar to css properties.

What is the standard in other components?